frontend/app/component/navbar/NavBar.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+import { IconUser } from "@kiesraad/icon";
+
+export const NavBar = ({ children }: { children: React.ReactNode }) => {
```

Comment:
Personally prefer function over arrow function like the other components.

Reply:
Changed in 105677be5f84a8008c4926b8a939fdf0e0a2ca62.